pyanaconda/rescue.py

```diff
@@ -149,13 +149,13 @@ def refresh(self, args=None):
         NormalTUISpoke.refresh(self, args)
 
         self._window += [TextWidget(_("The rescue environment will now attempt "
-                         "to find your Linux installation and mount it under "
-                         "the directory : %s.  You can then make any changes "
+                         "to find your Linux installation and\nmount it under "
+                         "the directory : %s.  You can then make any changes\n"
                          "required to your system.  Choose '1' to proceed with "
                          "this step.\nYou can choose to mount your file "
-                         "systems read-only instead of read-write by choosing "
+                         "systems read-only instead of read-write by\nchoosing "
                          "'2'.\nIf for some reason this process does not work "
-                         "choose '3' to skip directly to a shell.\n\n") % (iutil.getSysroot())), ""]
+                         "choose '3' to skip directly\nto a shell.\n\n") % (iutil.getSysroot())), ""]
```

Comment:
Please move the parts of the strings after ``\n`` to the next line so that the newlines are better visible.